website/source/docs/agent/config.html.md

```diff
@@ -152,6 +152,14 @@ nodes, unless otherwise specified:
   on server nodes. The value is a map of IP addresses and ports and supports
   the following keys:
   <br>
+  * `http`: The address to advertise for the HTTP interface. This should be
```

Comment:
Right above is where people are getting confused: "This configuration is only applicable on server nodes. The value is a map of IP addresses and ports and supports on server nodes."